### ISSUE TITLE
Unable to get property 'transition' of undefined or null reference

### ISSUE THREAD

**Issue by rhyek** [lifaon74/events-polyfill#1]
Hi. After doing `import 'events-polyfill'`, I get the following error in IE 11:

```
TypeError: Unable to get property 'transition' of undefined or null reference
   at EventListenerInterceptor.isSupportedTransitionEvent (http://localhost:3000/packages/modules.js?hash=6078294545688c872c4b6f56253c389881a374dc:1296:5)
   at EventListenerInterceptor.getSupportedEventType (http://localhost:3000/packages/modules.js?hash=6078294545688c872c4b6f56253c389881a374dc:1362:9)
   at add (http://localhost:3000/packages/modules.js?hash=6078294545688c872c4b6f56253c389881a374dc:1380:9)
   at target.prototype.addEventListener (http://localhost:3000/packages/modules.js?hash=6078294545688c872c4b6f56253c389881a374dc:1221:13)
   at ResizeObserverController.prototype.connect_ (http://localhost:3000/packages/modules.js?hash=6078294545688c872c4b6f56253c389881a374dc:78209:5)
   at ResizeObserverController.prototype.addObserver (http://localhost:3000/packages/modules.js?hash=6078294545688c872c4b6f56253c389881a374dc:78128:9)
   at ResizeObserverSPI.prototype.observe (http://localhost:3000/packages/modules.js?hash=6078294
```

**Comment by lifaon74** [comment]
I published a v2.0.3 could you try ?

The v2.X is intended to use only the required events polyfills (for smaller size). For example, if you only need to polyfill event listener options. You can require this specific file : require('src/ListenerOptions.js')

**Comment by rhyek** [comment]
Hey, @lifaon74, sorry. Been rather busy. I did try it out and got a different error. Really busy, couldn't take a screenshot, sorry. I will use the specific polyfill I need for now.

Thank you very much for trying to help :).
